tests/src/Kernel/Entity/GetEntityGroupsTest.php

```diff
@@ -53,32 +56,22 @@ class GetEntityGroupsTest extends KernelTestBase {
   protected $group2;
 
   /**
-   * The machine name of the group node type.
-   *
-   * @var string
-   */
-  protected $groupBundle;
-
-  /**
-   * The machine name of the group content node type.
-   *
-   * @var string
-   */
-  protected $groupContentBundle;
-
-  /**
    * {@inheritdoc}
    */
   protected function setUp() {
     parent::setUp();
 
-    $this->installConfig(['og']);
-    $this->installEntitySchema('og_membership');
-    $this->installEntitySchema('user');
-    $this->installEntitySchema('entity_test');
-    $this->installSchema('system', 'sequences');
+    // todo: check why this won't work with static variable.
+    $this->schemas = [
+      'config' => ['og'],
+      'entity' => ['og_membership', 'user', 'entity_test'],
+      'modules' => ['system' => ['sequences']],
+    ];
+
```

Comment:
This is not shorter really but changes from IDE integrated, autocomplete-able, ubiquitous methods to a proprietary mystery meat array. Strongly not recommended. (If you don't have an IDE with autocomplete, I recommend fixing that instead.)